### PR TITLE
Implement vertex coloring

### LINKED ISSUE #17: support per-vertex color

### PR DESCRIPTION
**Pull request by chuigda**
Closes #17

### CODE REVIEW

File: Cargo.toml
Comment (flomonster) [inline Cargo.toml]: I prefer to reserve this field for listing the lib's maintainers.
If you'd like to contribute more to this lib, I'd be delighted to add you as a maintainer.

File: src/scene/model/vertex.rs
Comment (flomonster) [inline src/scene/model/vertex.rs]:
```suggestion
            color: Zero::zero(),
```

**Review by flomonster — CHANGES_REQUESTED** [review]
Hi, thank you for your interest in the project and for your contribution.

I made a few suggestions before merging the branch.

File: src/scene/model/vertex.rs
Comment (chuigda) [reply src/scene/model/vertex.rs]: I found it more meaningful to use white (one) instead of transparent (zero) as default value here, since glTF specifies that the vertex color serves acts as an additional linear multiplier to base color.

Here I also tried `One::one`, but that trait `One` actually stands for "mathematical identity of that type" and is not implemented for vectors.

**Review by flomonster — APPROVED** [review]
LGTM

**Pull request merged**
